Adyen/CheckoutConfiguration/CheckoutBaseCallbacks.swift

```diff
@@ -27,3 +27,13 @@ package protocol CheckoutBaseCallbacks {
     
     var onComplete: CheckoutSuccessHandler? { get set }
 }
+
+package extension CheckoutBaseCallbacks {
+    
+    mutating func setBaseCallbacks(from configuration: CheckoutConfiguration) {
+        self.onSubmit = configuration.onSubmit
```

Comment:
I imagine this grows as we move forward. wdyt of an idea to store all callbacks in a dictionary with a key enumerating all supported callbacks? Then accessing and updating would be one line and there would be no `if let callback = self.callback else` anymore